app/models/post.rb

```diff
@@ -0,0 +1,7 @@
+class Post < ApplicationRecord
+  belongs_to :users
+  has_many :comments
+
+  validates :title, presence: true
+  validates :body, presence: true
+end
```

Comment:
- [ ] A post belongs to a particular user, kindly change the relationship from belongs_to:users to belongs_to :user

Reply:
Thanks so much for that good review, very clear, helped me understand where I had gone wrong and understand the topic better